pandas/core/resample.py

```diff
@@ -1025,7 +1025,10 @@ class TimeGrouper(Grouper):
     Use begin, end, nperiods to generate intervals that cannot be derived
     directly from the associated object
     """
-
+    _attributes = Grouper._attributes + ('closed', 'label', 'how',
+                                         'nperiods', 'fill_method', 'limit',
```

Comment:
can you leave out ``nperiods``, also ``fill_method`` and ``limit`` which are deprecated.

Reply:
you can actually completely remove ``nperiods`` as this is non-functional.